### ISSUE TITLE
Snapshot versions to include child taxa

### ISSUE THREAD

**Issue by m-r-c** [AtlasOfLivingAustralia/profile-hub#246]
1. When creating a snapshot version of a profile:
   1. Automatically create a snapshot version of all child taxa
   2. Comprise a list of the specific versions of all child taxa
   3. Include child taxa in the parent snapshot pdf
   4. Add an appendix listing the specific versions (with doi references) of the child taxa

**Comment by m-hope** [comment]
Also there needs to be a choice of inclusion of child taxa or not at time PDF created.

**Comment by ChrisPalmer1** [comment]
High priority

**Comment by RobinaSanderson** [comment]
Snapshot is: Archive of what the profile looked like before the last edit. 
Needs to be an option that can be turned on and off.

Middle priority agreed by eFlora team on 22/3/2017

**Comment by RobinaSanderson** [comment]
Does the asynchrous process for including children block the snapshot?

**Comment by RobinaSanderson** [comment]
Should the collection wide limit on the number of child taxa apply to this function?

**Comment by KevinThiele** [comment]
The ABRS guys will need to answer that question Robina. On one hand, snapshotting could usefully be restricted to a single profile - this is the state of this profile at this time. On the other hand, a snapshot of, say, a genus profile that includes all children gives more information - this is the state of this profile *and* these are all its children at this time. For this latter to be useful, we'd need to be able to include all of e.g. Acacia's children, which appears to be non-do-able. So I reckon it would be OK for snapshots to not include children at all, the information in the snapshot thus only fulfilling the first function.,

**Comment by ChrisPalmer1** [comment]
Yes, with the issues we've seen with including all child taxa when there are very many, I think it's probably best to restrict this function for now.
